src/main/java/seedu/address/model/Transcript.java

```diff
@@ -94,6 +95,76 @@ public void removeModule(Module key) {
         modules.remove(key);
     }
 
+    /**
+     * Return the current CAP
+     * @return current cap score
+     */
+    public double getCurrentCap() {
```

Comment:
I feel like the naming of both of these should be standardized. Either `getCurrentCapScore` and `calculateCapScore` (so we know explicitly at both levels of abstraction that we are returning a score, since the scores and grades thing may be confusing) or `getCurrentCap` and `calculateCap` (since CAP will always be a score so we make it implicit). But if you think the additional info is only appropriate for the lower level of abstraction then I think it's fine too.

Reply:
Oh just remembered CAP stands for Cumulative Average Point, so calling it `CapScore` is redundant right. And if we do need to be explicit elsewhere, use the term `Point` instead of `Score`.